README.md

```diff
@@ -19,21 +19,30 @@ React v16.8 or higher.
 ## Features
 ---
 
-The Purple Dot React Library currently only supports Purple Dot's Combined Checkout method. In this checkout method the Purple Dot backend will set the `enable oversell` option on products with a waitlist making them appear as available to buy. This library then works with your React components to provide the following:  
```

Comment:
This will read with a somehow negative sentiment to some merchants so we don't want to start with that.

Reply:
That's fine it's documentation, feel free to reword but its the same as starting the Readme with Minimum requirements

Reply:
Also we don't need to mentioned the term "combined", because 
- it's the only option we want to offer going forward
- it only makes sense internally. in most conversations with merchants we actually use the term "full bag" but that's equally confusing to the developer reading these docs.

Reply:
We've got 3 different types of checkout methods, and this library will need to support all 3. v1 supports separate bag only and before we publish 2.0 we'll need to add separate bag support back into this library

Reply:
Will it? I thought we said premium stores will always be on combined checkout going forward.